tests/web.rs

```diff
@@ -10,23 +10,29 @@ use web_sys::console;
 wasm_bindgen_test_configure!(run_in_browser);
 
 fn test_wallet() -> Result<WalletWrapper, String> {
-    let network = "testnet".to_string();
-    let descriptor = "wpkh([c258d2e4/84h/1h/0h]tpubDDYkZojQFQjht8Tm4jsS3iuEmKjTiEGjG6KnuFNKKJb5A6ZUCUZKdvLdSDWofKi4ToRCwb9poe1XdqfUnP4jaJjCB2Zwv11ZLgSbnZSNecE/0/*)".to_string();
-    let change_descriptor = "wpkh([c258d2e4/84h/1h/0h]tpubDDYkZojQFQjht8Tm4jsS3iuEmKjTiEGjG6KnuFNKKJb5A6ZUCUZKdvLdSDWofKi4ToRCwb9poe1XdqfUnP4jaJjCB2Zwv11ZLgSbnZSNecE/1/*)".to_string();
-    let esplora = "http://localhost:8010/proxy".to_string();
+    let network = "signet".to_string();
+    let descriptor = "tr([12071a7c/86'/1'/0']tpubDCaLkqfh67Qr7ZuRrUNrCYQ54sMjHfsJ4yQSGb3aBr1yqt3yXpamRBUwnGSnyNnxQYu7rqeBiPfw3mjBcFNX4ky2vhjj9bDrGstkfUbLB9T/0/*)#z3x5097m"
+    .to_string();
+    let change_descriptor = "tr([12071a7c/86'/1'/0']tpubDCaLkqfh67Qr7ZuRrUNrCYQ54sMjHfsJ4yQSGb3aBr1yqt3yXpamRBUwnGSnyNnxQYu7rqeBiPfw3mjBcFNX4ky2vhjj9bDrGstkfUbLB9T/1/*)#n9r4jswr"
+    .to_string();
+    let esplora = "https://mutinynet.com/api".to_string();
```

Comment:
great thanks I'll try with `mutinynet` as well. Like I said in previous PR, the tests work for me and I will try to test on the extension tomorrow and let you know. Please check the updated `main`.